src/utils/bytesParsers/formatBytes.ts

```diff
@@ -105,27 +69,64 @@ const formatToSignificantDigits = ({
         size = 'tb';
     }
 
-    return formatToSize({value, size, ...params});
+    return size;
+};
+
+interface FormatToSizeArgs {
+    value: number;
+    size?: BytesSizes;
+    precision?: number;
+}
+
+const formatToSize = ({value, size = 'mb', precision = 0}: FormatToSizeArgs) => {
+    const result = (Number(value) / sizes[size].value).toFixed(precision);
+
+    return formatNumber(result);
 };
 
-export type FormatBytesArgs = Omit<FormatToSizeArgs, 'value'> &
-    Omit<FormatToSignificantDigitsArgs, 'value'> & {
-        value: number | string | undefined;
-    };
+const addSizeLabel = (result: string, size: BytesSizes) => {
+    return result + ` ${sizes[size].label}`;
+};
+
+const addSpeedLabel = (result: string, size: BytesSizes) => {
+    return addSizeLabel(result, size) + i18n('perSecond');
+};
+
+export type FormatBytesArgs = Omit<FormatToSizeArgs, 'value'> & {
+    value: number | string | undefined | null;
+    withSpeedLabel?: boolean;
+    withSizeLabel?: boolean;
+    significantDigits?: number;
+};
 
 /**
  * @param significantDigits - number of digits above 3
  */
-export const formatBytes = ({value, size, ...params}: FormatBytesArgs) => {
+export const formatBytes = ({
+    value,
+    size,
+    withSpeedLabel = false,
```

Comment:
I meant a little bit different. I suggested not to have this props in this function: instead of passing this props let's call different functions - `formatSpeedBytes` and `formatSizeBytes`. Inside this two functions `formatBytes` should be called.

Reply:
In this case I wanted to create universal function to format bytes, so it was made intentionally. And for me additional `withSpeedLabel` and `withSizeLabel` params don't make it purpose unclear

Reply:
It's up to you! As for me - the less `if`-s - the better :)